Source/ACE.Entity/AceObject.cs

```diff
@@ -973,6 +973,34 @@ public object Clone()
             return ret;
         }
 
+        public object CopyClone(uint guid)
```

Comment:
Fixed - I had a brain freeze.   I need to be able to assign a new aceobjectid but I could do that with no duplicate code - just a new signature.